prov/efa/src/rxr/rxr_env.c

```diff
@@ -81,7 +82,7 @@ void rxr_env_param_get(void)
 {
 	if (getenv("FI_EFA_SHM_MAX_MEDIUM_SIZE")) {
 		fprintf(stderr,
-			"FI_EFA_SHM_MAX_MEDIUM_SIZE env variable detected! The use of this variable has been deprecated and as such execution cannot proceed.\n");
+			"FI_EFA_SHM_MAX_MEDIUM_SIZE env variable detected! The use of this variable has been deprecated and as such execution cannot proceed.n");
```

Comment:
Did you intend to remove the newline here?

Reply:
obe